src/dblib/dblib.c

```diff
@@ -1196,18 +1199,6 @@ tdsdbopen(LOGINREC * login, const char *server, int msdblib)
 		tdsdump_log(TDS_DBG_FUNC, "tdsdbopen(%p, %s, [%s])\n", login, server? server : "0x0", msdblib? "microsoft" : "sybase");
 	}
 
-	/*
-	 * Sybase supports the DSQUERY environment variable and falls back to "SYBASE" if server is NULL. 
-	 * Microsoft uses a NULL or "" server to indicate a local server.  
-	 * FIXME: support local server for win32.  
-	 */
-	if (!server && !msdblib) {
```

Comment:
Did you read the comment before deleting it and code? This is the expected behaviour, you are not supposed to change it for compatibility reasons.